mappings/net/minecraft/block/entity/MobSpawnerBlockEntity.mapping

```diff
@@ -13,4 +13,7 @@ CLASS net/minecraft/unmapped/C_qtiirvxr net/minecraft/block/entity/MobSpawnerBlo
 		ARG 1 pos
```

Comment:
hm, interesting point for a later PR: this class uses `MobSpawner` (the block entity's ID), but the block itself uses `Spawner` (the block/item's ID); I wonder if a convergence to `MonsterSpawner` would be better in the end;

You can ignore this message :P